spec/v1/envelope.md

```diff
@@ -1,23 +1,48 @@
 # Envelope layer specification
 
-Version: [DSSE v1.0]
-
-The Envelope is the outermost layer of the attestation, handling
-authentication and serialization.
+The Envelope is the outermost layer of the attestation, handling  serialization
+and authentication (via digital signatures).
 
 ## Schema
 
-The format and protocol are defined per [DSSE v1.0].
+The RECOMMENDED format and protocol for Envelopes are defined per [DSSE v1.0].
+Producers MAY use other signature methods and formats that meet the [ITE-5]
+specification:
+
+-   MUST support the inclusion of multiple signatures in a single envelope
+-   SHOULD include an authenticated payload type
+-   SHOULD avoid depending on canonicalization for security
+-   SHOULD support a hint indicating what signing key was used, i.e., a KEYID
+-   SHOULD NOT require the verifier to parse the payload before verifying
+-   SHOULD NOT require the inclusion of signing key algorithms in the signature
+
+### Alternative Envelope schemas
+
+-   The [Sigstore Bundle], while [supporting DSSE], is not currently [ITE-5]
+    compliant because it requires a _single signature_ in the envelope.[^1]
+-   The [COSE_Sign] structure is [ITE-5] compliant, whereas the `COSE_Sign1`
+    format that supports only one signer is NOT compliant.
 
 ## Fields
 
-The in-toto Attestation Framework has the following requirements for the
-standard DSSE fields.
+The in-toto Attestation Framework has the following general field requirements
+for an Envelope:
+
+-   `signature` (or equivalent) is REQUIRED and MUST be defined as an array.
+-   A `keyid` (or equivalent) SHOULD be included for each signing key used.
```

Comment:
It's also not obvious that a `keyid` is under a signature. Furthermore, it's missing the `sig` bit. Should we just point to the DSSE spec instead?

Reply:
I added some more text to clarify expectations a bit more. PTAL.